+++ nginx-length-hiding-filter-module/ngx_http_length_hiding_filter_module.c
@@ -274,7 +274,7 @@
         *s++ = base[ngx_random() % base_len];
         len--;
     }
-    s = ngx_copy(s, " -->", 4);
+    s = ngx_copy(s, " -->\n", 5);

     ngx_log_debug1(NGX_LOG_DEBUG_HTTP, r->connection->log, 0,
                    "http length hiding filter : str %s", d);
